server/chat-commands/info.ts

```diff
@@ -2644,6 +2645,32 @@ export const commands: ChatCommands = {
 	},
 	showhelp: [`/show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`],
 
+	regdate: 'registertime',
+	regtime: 'registertime',
+	async registertime(target, room, user) {
+		this.checkChat();
+		this.runBroadcast();
```

Comment:
Should this be restricted to only the user themself? It seems kinda creepy to broadcast the regdate of someone else...

Reply:
I mean, not really? It's already public - it would also be v useful for users who ask why they aren't autoconfirmed yet.

Reply:
...okay, I guess it's worth leaving it like this for now.